packages/paste-core/components/popover/__tests__/index.spec.tsx

```diff
@@ -245,6 +259,11 @@ describe('Popover', () => {
         </Theme.Provider>
       );
 
+      const popoverButton = screen.getByRole('button');
+      await waitFor(() => {
+        userEvent.click(popoverButton);
+      });
+
```

Comment:
Again, preset visible causes an unwrapped state update. We can achieve the same test, by just clicking the thing and using the test utils to handle it all safely.